backend/coreapp/models.py

```diff
@@ -60,11 +78,12 @@ class Scratch(models.Model):
     diff_label = models.CharField(max_length=512, blank=True, null=True)
     score = models.IntegerField(default=-1)
     max_score = models.IntegerField(default=-1)
-    parent = models.ForeignKey("self", null=True, blank=True, on_delete=models.CASCADE)
-    owner = models.ForeignKey(Profile, null=True, blank=True, on_delete=models.SET_NULL)
+    parent = models.ForeignKey("self", null=True, blank=True, on_delete=models.CASCADE) # XXX: should be SET_NULL?
+    owner = models.ForeignKey(Profile, null=True, blank=True, on_delete=models.SET_NULL) # XXX: should be CASCADE?
```

Comment:
If the owner is deleted, I think we can just set owner to null as well...

Reply:
I guess I was thinking that deleting your account would also imply you want all data deleted